types/types.go

```diff
@@ -1,10 +1,23 @@
 package types
 
+/*type LogLevel int
```

Comment:
May as well remove these unless you are like OMG I WILL FIGURE THIS OUT

Reply:
> May as well remove these unless you are like OMG I WILL FIGURE THIS OUT

Done 40c2917